examples/components/Radar.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <v-wedge :config='config'></v-wedge>
+</template>
+
+<script>
+import {Werge} from 'konva'
+export default {
+  name: 'Radar',
+  extends: Werge,
+  props: {
+    config: {
+      type: Object,
+      default: {
+        direction: 0
+      }
+    }
+  },
+  watch: {
+    'config.angle': function (newVal, oldVal) {
+      this.handleAngleChange(true);
+    },
+    'config.direction': function (newVal, oldVal) {
+      this.handleDirectionChange(false);
+    }
+  },
+  methods: {
+    handleAngleChange(angle) {
+      this.config.rotation = this.config.direction - this.config.angle / 2.0
+    },
+    handleDirectionChange(angle) {
+      // todo need better solution
+      const shape = this.$parent.getStage().children[0]
```

Comment:
Then use `const shape = this.$refs.wedge.getStage()` so you don't have to care about parent component

Reply:
wow awesome, Thanks a lot! The reason why I did pull request is I wanted help :) and some correction of my code. I am new in vuejs